pylds/vector_fields.py

```diff
@@ -181,5 +181,99 @@ def forcing(t, u, perturbation_params = [1, 0.15, 0.5]):
     
     return perturbation
 
+def HenonHeiles_vector_field(t, u, PARAMETERS = None):
+    """
+    Returns 2D Henon-Heiles vector field at time t, for an array of points in phase space.
+    Number of model parameters: 0 . PARAMETERS = [wx, wy, delta]
```

Comment:
This is confusing. If the system has 0 parameters, why are there three?

Reply:
Yeah, for most functions in `vector_fields.oy` I define `PARAMETERS` having a common template in mind, which can be tweaked later on by others, that's why `None` is defined provisionally. 

docstrings are outdated, that why three parameters are showing up. I will update it shortly.

Reply:
Fixed!